components/analytics/system_analytics/system_analytics.tsx

```diff
@@ -313,7 +322,7 @@ export default class SystemAnalytics extends React.PureComponent {
                     />
                 }
                 icon='fa-globe'
-                count={stats[StatTypes.TOTAL_PUBLIC_CHANNELS] + stats[StatTypes.TOTAL_PRIVATE_GROUPS]}
+                count={this.getStatValue(stats[StatTypes.TOTAL_PUBLIC_CHANNELS]) + this.getStatValue(stats[StatTypes.TOTAL_PRIVATE_GROUPS])}
             />
         );
 
```

Comment:
@Willyfrog 
WIth that i have to do something like this to resolve this ?

        const totalPublicChannelsCount = this.getStatValue(stats[StatTypes.TOTAL_PUBLIC_CHANNELS]);
        const totalPrivateGroupsCount = this.getStatValue(stats[StatTypes.TOTAL_PRIVATE_GROUPS]);
        const totalChannelCount = () => {
            if (totalPublicChannelsCount && totalPrivateGroupsCount) {
                return totalPublicChannelsCount + totalPrivateGroupsCount;
            } else if (!totalPublicChannelsCount && totalPrivateGroupsCount) {
                return totalPrivateGroupsCount;
            } else if (totalPublicChannelsCount && !totalPrivateGroupsCount) {
                return totalPublicChannelsCount;
            }
            return undefined;
        };
        const channelCount = (
            <StatisticCount
                id='totalChannels'
                title={
                    <FormattedMessage
                        id='analytics.system.totalChannels'
                        defaultMessage='Total Channels'
                    />
                }
                icon='fa-globe'
                count={totalChannelCount()}
            />
        );

Reply:
yes, and also make sure that `StatisticCount` can handle undefined

Reply:
Alright will do

Reply:
StatisticCount does handle undefined count values.

I have pushed new changes.